app/frontend/Services/WebPdfViewer.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using ClientApp.Components;
```

Comment:
This should be removed from here, and instead defined as part of `GlobalUsings`.

Reply:
Fixed.